picard/ui/playertoolbar.py

```diff
@@ -370,7 +377,10 @@ def __init__(self, parent, volume):
         icon = self.style().standardIcon(QtWidgets.QStyle.SP_MediaVolume)
         self.setIcon(icon)
         self.set_volume(volume)
-        self.setText('100%')  # Only for initial min width calculation
+        margins = self.getContentsMargins()
+        button_margin = self.style().pixelMetric(QtWidgets.QStyle.PM_ButtonMargin)
+        min_width = get_text_width(self.font(), '888%')
```

Comment:
You should use same format on display and width calculation: _('%d%%') rather than hardcoded '888%'

Reply:
Fixed as part of https://github.com/metabrainz/picard/pull/1231/commits/164c32ff0131de30f5a97634f1a97e5de157f46c